lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -47,6 +47,7 @@
             {'path': '/usr/sbin/sorcery', 'name': 'sorcery'},
             {'path': '/usr/bin/installp', 'name': 'installp'},
             {'path': '/QOpenSys/pkgs/bin/yum', 'name': 'yum'},
+            {'path': '/data/data/com.termux/files/usr/bin/apt', 'name': 'apt'},
```

Comment:
Do you know if this path is reliable, or depends on the configuration of the Android device?

Reply:
I know you can move, or explicitly install apps onto other disks, such as an extra SD card. So would that make the path different, or is this some how a chrooted path inside the app that always exists?

Reply:
According to their wiki, https://github.com/termux/termux-packages/wiki/Termux-file-system-layout, it seems to be a stable path. Termux exports a `${PREFIX}` variable, but it's not configurable according to [this](https://wiki.termux.com/wiki/Differences_from_Linux):

> The root file system and user home directory are located in a private application data directory which lives on the /data partition. Paths to these directories are exposed as $PREFIX and $HOME respectively.

> You cannot move $PREFIX to another location because all programs expect that $PREFIX will not be changed. Additionally, you cannot have binaries, symlinks and other files from $PREFIX on sdcard because that file system does not support unix permissions, symlinks, sockets, etc... 

As an aside, thanks for all of the work on Ansible. This tool has changed my life :+1: !